packages/react-async/src/hooks.ts

```diff
@@ -80,6 +80,10 @@ export function useAsync<T>(
     try {
       const resolved = await resolver.resolve();
 
+      if (resolver.requireError) {
+        throw resolver.requireError.error;
```

Comment:
I've never fully understood webpack's behaviour during SSR breakages, so I'm fuzzy on if this is a behavioural change or not 😅 

My understanding of old behaviour is:
* The first failing webpack `require` is silently swallowed 
* `null` is returned to the caller
* The caller renders nothing for this component (and all children), and HTML contains a shallow component tree

So does this:
1. Behave exactly as before
2. Display an error for the first failing request, then shallow render subsequent requests
3. Throw on every request until a rollback

I think it's _2_, but may be completely wrong about everything 😸